### ISSUE TITLE
blocked when using SendAndConfirmTransactionWithOpts

### ISSUE THREAD

**Issue by boyi** [gagliardetto/solana-go#91]
Sometimes the websocket in SendAndConfirmTransactionWithOpts just blocked and did't return anything. Even if I pass c ContextWithDeadline as the context of the wsClient. 

Any one know how to solve this issue?

**Comment by gagliardetto** [comment]
That probably means the transaction was never executed/never reached a validator.

Do you get an error somewhere? Are you on mainnet? Devnet?

**Comment by boyi** [comment]
No, I didn't get any error. Should I check the status before I start the ws listening?
